apps/app/next.config.js

```diff
@@ -7,7 +7,6 @@ const withTM = require('next-transpile-modules')([
   '@devlaunchers/site-projects',
   '@devlaunchers/dev-recruiters',
   '@devlaunchers/website',
-  '@devlaunchers/user-profile',
```

Comment:
why remove this? I think it's needed

Reply:
maybe it was part of a conflict but definitely needed to be back

Reply:
` '@devlaunchers/user-profile'` is duplicated, so i deleted one of them. If you expand it you'll see it on top